GoogleForCreators/web-stories-wp
# Design System Storybook - Icons

## Summary

Adds all unique icons from figma 'Foundation' (formerly 'Earth') section of design system as SVGs and demos in storybook

## Relevant Technical Choices

- Organized icons in the same style as the[ material icons](https://material.io/resources/icons/?style=baseline) in the hopes that having some subdirectories makes navigating icon choices easier for people joining the project later on by giving them a little bit more context. 
- Any icon that has only 1 color involved, I swapped to `currentColor` to make them easier to manipulate 
- Any icon that is repeated in figma and just rotated, I only placed in once (tried to default to left for consistency) 

## To-do
- N/A

## User-facing changes
- No changes, icons just visible in storybook/designSystem 

## Testing Instructions

- Spin up storybook and see that you have 72 icons loading when you visit designSystem/Icons/All Icons 
<img width="1198" alt="Screen Shot 2020-10-20 at 4 25 16 PM" src="https://user-images.githubusercontent.com/10720454/96655121-96b82180-12f1-11eb-961f-4a0be0ec68c8.png">


---



Fixes #
